### PR TITLE
Lookaround

### PR DESCRIPTION
**Pull request by pakonro**
… nao leaves penalised or initial state

## Introduced Changes

*Describe here what this pull request does*

Fixes #

## ToDo / Known Issues

*If this is a WIP describe which problems are to be fixed.*

## Ideas for Next Iterations (Not This PR)

*If there are some improvements that could be done in a next iteration, describe them here.*

## How to Test

*Describe how to test your changes. (For the reviewer)*

### CODE REVIEW

File: etc/configuration/default.json [hunk 1]
```diff
@@ -853,7 +853,8 @@
     "look_action": {
       "angle_threshold": 0.95,
       "distance_threshold": 3.0
-    }
+    },
```
Comment (h3ndrk) [inline etc/configuration/default.json:856]: Fucking JSON... 😒

File: crates/types/src/configuration.rs [hunk 2]
```diff
@@ -83,6 +83,7 @@ pub struct Behavior {
     pub walk_and_stand: WalkAndStand,
     pub search: Search,
     pub look_action: LookAction,
+    pub initial_lookaround_duration: f32,
```
[the remark this answers is not in the document]
Comment (pakonro) [reply crates/types/src/configuration.rs]: Done

File: crates/control/src/behavior/node.rs [hunk 3]
```diff
@@ -82,6 +101,14 @@ impl Behavior {
             Action::Stand,
         ];
 
+        if let Some(active_since) = self.active_since {
+            if now.duration_since(active_since).unwrap().as_secs_f32()
```
[the remark this answers is not in the document]
Comment (pakonro) [reply crates/control/src/behavior/node.rs]: Done

[the remark this answers is not in the document]
Comment (pakonro) [reply crates/control/src/behavior/node.rs]: Done

**Review by okiwi6 — CHANGES_REQUESTED** [review]
Nice, the changes look good to me. The CI is still failing, please do a `cargo fmt` on your code to format. Also `behavior_simulator` is unable to build since you added `cycle_time` in the behavior context but not in the behavior_simulator context (you can check that it is not working by running `cargo build --manifest-path tools/behavior_simulator/Cargo.toml`.
On a more general note, please fill in the PR description next time :)

File: etc/configuration/default.json [hunk 4]
```diff
@@ -901,12 +901,17 @@
     },
     "look_action": {
       "angle_threshold": 0.95,
+
```
Comment (okiwi6) [inline etc/configuration/default.json:904]: please remove newline

**Review by okiwi6 — CHANGES_REQUESTED** [review]

**Pull request merged**